### PR TITLE
Calculate default guide rates using the well potentials

### PR DESCRIPTION
**Pull request by totto82**
The default guide rates are calculated using the well potentials.
The well potentials are calculated in the simulator and given as input
to the wellsManager.

### CODE REVIEW

File: opm/core/wells/WellsManager.cpp [hunk 1]
```diff
@@ -330,11 +331,12 @@ namespace Opm
                                const double* permeability)
         : w_(0), is_parallel_run_(false)
     {
+        std::vector<double> well_potensials;
```
Comment (atgeirr) [inline opm/core/wells/WellsManager.cpp]: Typo: potensials -> potentials

File: opm/core/wells/WellsManager.hpp [hunk 2]
```diff
@@ -86,7 +86,8 @@ namespace Opm
                      const F2C& f2c,
                      FC begin_face_centroids,
                      const double* permeability,
-                     bool is_parallel_run=false);
+                     bool is_parallel_run=false,
+                     const std::vector<double> well_potentials={});
```
Comment (atgeirr) [inline opm/core/wells/WellsManager.hpp]: Should be a reference argument (also in other methods).

**Comment by atgeirr** [comment]
Looks good, will merge!

**Comment by totto82** [comment]
Doesn't this PR need to be merged together with OPM/opm-autodiff#627. The wells_manager interface is changed.

**Comment by atgeirr** [comment]
> Doesn't this PR need to be merged together with OPM/opm-autodiff#627.

I think the default parameter allows for this to be merged separately. In case I failed I intend to merge the other PR after a build check very soon.

**Comment by totto82** [comment]
You  are right, I forgot I added a default.

**Pull request merged**